src/index.js

```diff
@@ -6,4 +6,9 @@ export default [
   'The leak is minor, we’ll just add more RAM later.',
   'Looks like it’s all working fine',
   'We’ll fix it later',
+  'If it works it works.',
+  'Let’s build our own framework for it.',
+  'This plugin should cover all our needs.',
```

Comment:
Building your own thing is a bad idea. Not building your own thing and using ready-made plugins for everything is also a bad idea. The truth is somewhere in between and depends on the project imho.